cob_base_drive_chain/ros/src/cob_base_drive_chain.cpp

```diff
@@ -500,7 +500,7 @@ class NodeClass
 		{
 			if(m_bisInitialized)
 			{
-				ROS_DEBUG("Service callback reset");
+				ROS_DEBUG("Service callback /base_controller/recover");
```

Comment:
@ipa-mig-tk @ipa-mig 
even if this is only ROS_DEBUG, can we get rid of the hardcoded namespace ```/base_controller``` in the text - as this is going to change anyway.